connectivity/drivers/mbedtls/TARGET_Cypress/TARGET_MXCRYPTO/ecp_alt.c

```diff
@@ -2619,6 +2619,44 @@ int mbedtls_ecp_read_key( mbedtls_ecp_group_id grp_id, mbedtls_ecp_keypair *key,
     return( ret );
 }
 
+/*
+ * Write a private key.
+ */
+int mbedtls_ecp_write_key( mbedtls_ecp_keypair *key,
```

Comment:
This was copied from https://github.com/ARMmbed/mbedtls/blob/523f0554b6cdc7ace5d360885c3f5bbcc73ec0e8/library/ecp.c#L3211-L3247 and I've adjusted to the macro names from `MBEDTLS_ECP_MONTGOMERY_ENABLED` & `MBEDTLS_ECP_SHORT_WEIERSTRASS_ENABLED` to `ECP_MONTGOMERY` & `ECP_SHORTWEIERSTRASS` - the latter names are being used in Cypress's driver.